bin/generate.ts

```diff
@@ -53,9 +77,14 @@ headers += `#define LLHTTP_VERSION_MINOR ${version.minor}\n`;
 headers += `#define LLHTTP_VERSION_PATCH ${version.patch}\n`;
 headers += '\n';
 
+headers += '#ifndef LLHTTP_STRICT_MODE\n';
+headers += '# define LLHTTP_STRICT_MODE 1\n';
```

Comment:
`#define` space typo?

Reply:
Not really. We are doing this in Node, libuv, and http-parser. It is sort of an indent for nested defines.